api/src/api/organizations_v1/organization_schemas.py

```diff
@@ -335,3 +337,105 @@ class OrganizationIgnoreLegacyUserRequestSchema(Schema):
 
 class OrganizationIgnoreLegacyUserResponseSchema(AbstractResponseSchema):
     data = fields.MixinField(metadata={"example": None})
+
+
+class LegacyUserDataSchema(Schema):
+    """Schema for individual legacy user data"""
+
+    email = fields.String(
+        required=True,
+        metadata={"description": "Legacy user email address", "example": "user@example.com"},
+    )
+    full_name = fields.String(
+        required=True,
+        metadata={"description": "Legacy user full name", "example": "John Doe"},
+    )
+    status = fields.String(
+        required=True,
+        metadata={
+            "description": "Status of the legacy user relative to the organization",
+            "example": "available",
+        },
+    )
+
+
+class LegacyUserFilterSchema(Schema):
+    """Schema for filtering legacy users by various criteria"""
+
+    status = fields.Nested(
+        StrSearchSchemaBuilder("LegacyUserStatusFilterSchema")
+        .with_one_of(allowed_values=LegacyUserStatus, example="available")
+        .build(),
+        allow_none=True,
+        metadata={"description": "Filter legacy users by status"},
+    )
+
+
+# Custom pagination schema with defaults: page_size=10, page_offset=1
+LegacyUserSortOrderSchema = Schema.from_dict(
+    {
+        "order_by": fields.String(
+            validate=[validators.OneOf(["email", "full_name", "created_date"])],
+            required=True,
+            metadata={"description": "The field to sort the response by"},
+        ),
+        "sort_direction": fields.Enum(
+            SortDirection,
+            required=True,
+            metadata={"description": "Whether to sort the response ascending or descending"},
+        ),
+    },
+    name="LegacyUserSortOrderSchema",
+)
+
+LegacyUserPaginationSchema = BasePaginationSchema.from_dict(
```

Comment:
Why build this instead of using the `generate_pagination_schema`? Except for changing the default, I don't see anything else that is different, and if we want to be able to set a default, can just adjust the function to allow that as an optional parameter.